user/src/pages/movies/Movies.jsx

```diff
@@ -0,0 +1,66 @@
+import Header from '../../components/Header/Header';
+import Pagination from '../../components/Pagination/Pagination'
+
+const Movies = ({movies, totalMovies, selectedPage, setSelectedPage}) => {
+  const category_list = (movie) => {
+    return movie.categories.join(', ')
+  }
+  return (
+    <>
+      <Header />
+      {movies && movies.length ? (
+        <div className="container">
+          <div  className="row">
+            {movies.map((movie) => (
+                <div key={movie.id} className="col-lg-4 col-md-4 col-sm-8 col-xs-16 padding-right-30 padding-left-30 padding-bottom-30">
```

Comment:
bootstrap có col-12, mình dùng luôn không cần định nghĩa thêm col-16 nữa anh, cái web này của beta họ dùng bootstrap bản cũ nên mới có col-16 ấy
```suggestion
                <div key={movie.id} className="col-md-4 col-sm-8 col-xs-12 padding-right-30 padding-left-30 padding-bottom-30">
```